### PR TITLE
v1.30.2

### PR DESCRIPTION
**Pull request by njuettner**

### CODE REVIEW

File: helm/azuredisk-csi-driver-app/templates/_helpers.tpl [hunk 1]
```diff
@@ -11,10 +11,21 @@
...
-helm.sh/chart: {{ include "chart" . | quote }}
-application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+labels:
+  app.kubernetes.io/instance: "{{ .Release.Name }}"
+  app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+  app.kubernetes.io/name: "{{ template "azuredisk.name" . }}"
+  app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
+  helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+  application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+{{- end -}}
```
Comment (fiunchinho) [inline helm/azuredisk-csi-driver-app/templates/_helpers.tpl:21]: This looks wrong. Isn't `labels` already in the manifest so we don't have to repeat it here?

File: helm/azuredisk-csi-driver-app/templates/_helpers.tpl [hunk 2]
```diff
@@ -11,10 +11,21 @@
 
 {{/* labels for helm resources */}}
 {{- define "azuredisk.labels" -}}
-app.kubernetes.io/instance: "{{ .Release.Name }}"
-app.kubernetes.io/managed-by: "{{ .Release.Service }}"
-app.kubernetes.io/name: "{{ template "azuredisk.name" . }}"
-app.kubernetes.io/version: "{{ .Chart.AppVersion }}"
-helm.sh/chart: {{ include "chart" . | quote }}
-application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+labels:
```
Comment (fiunchinho) [inline helm/azuredisk-csi-driver-app/templates/_helpers.tpl:14]: it feels weird to include the `labels` keyword inside the helper, I don't think we do it like that anywhere else. Do we need to put it inside the helper?

Comment (njuettner) [reply helm/azuredisk-csi-driver-app/templates/_helpers.tpl:14]: Yes, you're right. I wanted to open the PR as a draft but noticed it too late 🙈. I wanted to see the diff easier. 

I'll change that for sure 👍🏻. Will let you know when it's finished. Sorry for pinging you too early 😬

**Comment by njuettner** [comment]
Tested in a Azure cluster, from my point it looks good now 🙂

**Review by Gacko — APPROVED** [review]

**Review by fiunchinho — APPROVED** [review]

**Pull request merged**